### PR TITLE
[Spec] Fix the buildbreak issue for VD

### PR DESCRIPTION
**Pull request by again4you**
* Disable OpenVino
* Omit the test case execution & testcase summary

Signed-off-by: Sangjung Woo <sangjung.woo@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -88,6 +88,7 @@
 %define		protobuf_support 0
 %define		python3_support 0
 %define		mvncsdk2_support 0
+%define		openvino_support 0
```
Comment (wooksong) [inline packaging/nnstreamer.spec:91]: Checked.

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -673,14 +674,14 @@ export NNSTREAMER_CONVERTERS=${NNSTREAMER_BUILD_ROOT_PATH}/ext/nnstreamer/tensor
...
 
-%if %{with tizen}
-    bash %{test_script} ./tests/tizen_nnfw_runtime/unittest_nnfw_runtime_raw
-    bash %{test_script} ./tests/tizen_sensor/unittest_tizen_sensor
-%endif #if tizen
 %if 0%{?unit_test}
     bash %{test_script} ./tests
     bash %{test_script} ./tests/cpp_methods
     bash %{test_script} ./tests/nnstreamer_filter_extensions_common
+%if %{with tizen}
```
Comment (wooksong) [inline packaging/nnstreamer.spec:681]: Q: When is this flag enabled?

File: packaging/nnstreamer.spec [hunk 3]
```diff
@@ -673,14 +674,14 @@ export NNSTREAMER_CONVERTERS=${NNSTREAMER_BUILD_ROOT_PATH}/ext/nnstreamer/tensor
 
 %define test_script $(pwd)/packaging/run_unittests_binaries.sh
 
-%if %{with tizen}
```
Comment (kparichay) [inline packaging/nnstreamer.spec:676]: let change this condition to exclude the tv profile so as to retain the original behavior of running these unittests on all architectures on our CI as discussed offline.

Comment (again4you) [reply packaging/nnstreamer.spec:676]: Updated.

File: packaging/nnstreamer.spec (hunk 2 above)
Comment (again4you) [reply packaging/nnstreamer.spec]: According to the PK's opinion and considering our CI, I update the condition as below.
```cpp
%if ( %{with tizen} && "%{?profile}" != "tv" )
```

**Review by wooksong — APPROVED** [review]
LGTM!

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**